ecs/s3.conf

```diff
@@ -0,0 +1,14 @@
+@INCLUDE /ecs/service.conf
+@INCLUDE /ecs/input-forward.conf
+@INCLUDE /ecs/ecs-metadata.conf
+
+[OUTPUT]
+    Name s3
+    Match *
+    bucket ${BUCKET}
+    region ${S3_REGION}
+    total_file_size 1M
+    upload_timeout 1m
+    use_put_object On
+    s3_key_format  /$TAG[0]/${TASK_ID}/%Y/%m/%d/%H/%M/%S
```

Comment:
S3 key should also have cluster

I think it should go from least specific, to most specific, so:

`/cluster/task ID/container name`